py/objint_mpz.c

```diff
@@ -436,11 +438,27 @@ mp_int_t mp_obj_int_get_checked(mp_const_obj_t self_in) {
             return value;
         } else {
             // overflow
-            mp_raise_msg(&mp_type_OverflowError, translate("overflow converting long int to machine word"));
+            mp_raise_msg(&mp_type_OverflowError, MP_ERROR_TEXT("overflow converting long int to machine word"));
         }
     }
 }
 
+mp_uint_t mp_obj_int_get_uint_checked(mp_const_obj_t self_in) {
```

Comment:
There are places we can this `mp_obj_int_get_uint_checked()`

Reply:
Where we can use this? I can't think of any immediately. Can we do that in a follow up?

Reply:
Oh, I don't mean we have to use this now. But I recently was working on som.e code that required positive arguments, and to have this checked automatically would be nice